rest-api-spec/test/indices.get_aliases/10_basic.yaml

```diff
@@ -202,7 +202,7 @@ setup:
 "Getting alias on an non-existent index should return 404":
 
   - skip:
-        version: 1 - 999
+        version: 0 - 999
```

Comment:
I got rid of the harry custom version parsing code here, but I just realized the integer versions were not quite what I thought.  I'll push a fix here soon (and this example will change to "1.0.0.Beta1 - ")